src/core_plugins/kibana/public/management/sections/indices/_create.js

```diff
@@ -247,6 +254,18 @@ uiModules.get('apps/management')
     }, notify.fatal);
   }
 
+  function updateFieldListAndSetTimeField(results, seedField) {
+    updateFieldList(results);
+
+    if (!results.dateFields.length) {
+      return;
+    }
+
+    const matchingTimeField = results.dateFields.find(field => field.name === seedField);
+    const defaultTimeField = results.dateFields[0];
+    index.timeField = matchingTimeField ? matchingTimeField : defaultTimeField;
```

Comment:
Likewise, I would wonder why we're pulling out and assigning `matchingTimeField`? Wouldn't we want to assign the field we're calling the function with? It seems like `matchingTimeField` is really just being used to make sure the time field we wish to set is actually available... so maybe this should just be:

```javascript
index.timeField = isTimeFieldAvailable ? timeField : defaultTimeField;
```